scrapers/scrape_ju.py

```diff
@@ -26,6 +26,18 @@
         rows.append(data)
 
     if rows:
+        # sanitize data:
+        # 2020-12-04 contains 'Non communiqué' entries, skip them for now
+        for row in rows:
+            if not row.get('Date') or row.get('Date') == 'Date':
+                continue
+            if not sc.int_or_word(row.get('Nombre de cas actuellement hospitalisés')):
+                row['Nombre de cas actuellement hospitalisés'] = ''
+            if not sc.int_or_word(row.get('Nombre de cas actuellement en soins intensifs')):
+                row['Nombre de cas actuellement en soins intensifs'] = ''
+            if not sc.int_or_word(row.get('Nombre de nouveaux décès')):
+                row['Nombre de nouveaux décès'] = ''
+
```

Comment:
Instead of creating an extra for loop, I would add a method called something like `sanitize_row` where you could make those checks.
And instead of using `sc.int_or_word` I would use `sc.represents_int`, which should cover these cases.

Reply:
Good point, should be addressed in c0597ee.